neuromation/api/admin.py

```diff
@@ -38,6 +38,26 @@ def __init__(self, core: _Core, config: Config) -> None:
             res = await resp.json()
             return [_cluster_user_from_api(payload) for payload in res]
 
+    async def add_cluster_user(
+        self, cluster_name: str, user_name: str, role: str
+    ) -> _ClusterUser:
+        role_type = _ClusterUserRoleType(role)
+        user = _ClusterUser(user_name, role_type)
+        url = self._config.admin_url / "clusters" / cluster_name / "users"
+        payload = {"user_name": user_name, "role": role}
+        auth = await self._config._api_auth()
+
+        async with self._core.request("POST", url, json=payload, auth=auth) as resp:
+            await resp.json()
```

Comment:
Loading and parsing a JSON is not needed if the value is ignored.

Or maybe use `_cluster_user_from_api()` for creating the returning value?

Reply:
Probably will do the 2nd one.